index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Movie Reviews</title>
+    <link rel="stylesheet" href="./styles/movie-card.css" />
+  </head>
+  <body>
+    <div class="movies-container" id="movies-container"></div>
+    <script type="module" src="./scripts/movie-card.js"></script>
```

Comment:
The `type="module"` is not necessary in your case. But you can keep it; it's up to you.